tf/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  flake-utils,
+  nixpkgs,
+  terranix,
+  ...
+}:
+
+flake-utils.lib.eachDefaultSystem (
+  system:
+
+  let
+    pkgs = nixpkgs.legacyPackages.${system};
+
+    opentofu = pkgs.opentofu.withPlugins (plugins: [
+      plugins.hcloud
+      plugins.netlify
+    ]);
```

Comment:
These plugins are registered under `libexec/terraform-providers/registry.terraform.io`, but opentofu looks for `registry.opentofu.org`. Additionally, the netlify provider you added isn't `netlify/netlify` but rather `AegirHealth/netlify`

Reply:
This *should* be avoided by including the registry name in the source reference like I setup [here](https://github.com/PrismLauncher/infra/blob/4b55aff847528df938db21f3cdf3035d639291b2/tf/modules/providers.nix#L34)...not sure why it's still trying to look for things off the OpenTofu registry when it's explicitly set